### PR TITLE
[#104945364] Fix --as-group and --as-role support in conjur policy load

### PR DESCRIPTION
**Pull request by apotterri**
Also, make sure bare resources created in a policy are owned by the policy.

### CODE REVIEW

**Comment by apotterri** [comment]
Also delivers [#104953136]

**Comment by apotterri** [comment]
Maybe a problem with this implementation, need some more tests to be sure.

File: lib/conjur/dsl/runner.rb [hunk 1]
```diff
@@ -194,6 +194,13 @@ def find_or_create(type, id, options, &block)
 
         unless (obj = api.send(find_method, id)) && obj.exists?
           options = expand_options(options)
+
+          # create_resource and create_role expect :acting_as to
+          # specify the "owning" role.
+          if create_method == :create_resource || create_method == :create_role
```
Comment (kgilpin) [inline lib/conjur/dsl/runner.rb]: Looks right

**Comment by apotterri** [comment]
@kgilpin ok, I feel better about this now.

**Pull request merged**